test/e2e/storage/testsuites/testdriver.go

```diff
@@ -88,6 +88,9 @@ type PreprovisionedPVTestDriver interface {
 type DynamicPVTestDriver interface {
 	TestDriver
 	// GetDynamicProvisionStorageClass returns a StorageClass dynamic provision Persistent Volume.
+	// The StorageClass must be created in the current test's namespace and have
+	// a unique name inside that namespace because GetDynamicProvisionStorageClass might
+	// be called more than once per test.
```

Comment:
BTW, I got curious why we now have this requirement. It found that it was introduced two months ago when implementing the multi-attach test cases, but then wasn't treated as the API change for the TestDriver interface but rather got handled only in some of its implementations (those that use `GetStorageClass`):
https://github.com/kubernetes/kubernetes/commit/b4c88acec63ebd678d2e5a7a3db1b6da22d39b96